wiki/utxos.md

```diff
@@ -138,21 +138,27 @@ Think of it as if your coins were part of a "raffle" and your UTXOs were a chunk
 of "tickets". No matter how you segment your "tickets" (total GRC), you still 
 have the same odds of "winning" (staking).
 
-## Reward Coinbase Transaction 
+## Reward Coinstake Transaction 
 
 The UTXO that staked is used up and a new special transaction 
 is formed to send rewards. This special transaction uses your existing 
-UTXO and something called a coinbase[^2] as inputs to send you your rewards 
-for staking. Coinbases are where the new coins come from. 
-They are also how the very first UTXOs on the network were made
+UTXO and something called a coinstake as inputs to send you your rewards 
```

Comment:
This is confusing to me. The staked input is the (sole) input to the coinstake transaction. The way this is worded make it sound like the coinstake is a separate input. The coinstake is the transaction.

Reply:
Oh wait I didn't realize that. With coinbases, the coinbase is the input. Looked at gridcoinstats.eu which showed coin generation as if it was an input in a coinstake transaction, so I thought it was the same. Looking closer I see a coinstake just sends more than the UTXO's size to create new coins

Reply:
@jamescowens is that any better?